src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -104,12 +104,11 @@ void Notifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const std::st
     int          port;
     std::string  uriPath;
     std::string  protocol;
-    
+    std::string  initialUrl = url;
```

Comment:
I took for granted that url is destroyed in parseUrl.
Perhaps it isn't ...

If it is not destroyed, then there is no need to copy it, of course.